client/ctclient/ctclient.go

```diff
@@ -508,33 +500,17 @@ func main() {
 			glog.Exitf("Multiple logs with name like %q found in loglist: %s", *logName, strings.Join(logNames, ","))
 		}
 		uri = "https://" + logs[0].URL
-		if *useDNS {
-			dns = logs[0].DNSAPIEndpoint
-		}
 		if opts.PublicKey == "" {
 			opts.PublicKeyDER = logs[0].Key
 		}
 	}
-	if *useDNS && dns == "" {
-		glog.Exit("DNS access requested (with --dns) but no DNS base name known")
-	}
 
 	var err error
 	var logClient *client.LogClient
 	var checkClient client.CheckLogClient
-	if dns != "" {
-		if *dnsServer != "" {
-			glog.V(1).Infof("Use DNS server at %s for basename %s", *dnsServer, dns)
-			checkClient, err = dnsclient.NewForNameServer(dns, opts, *dnsServer)
-		} else {
-			glog.V(1).Infof("Use system DNS resolver for basename %s", dns)
-			checkClient, err = dnsclient.New(dns, opts)
-		}
-	} else {
-		glog.V(1).Infof("Use CT log at %s", uri)
-		logClient, err = client.New(uri, httpClient, opts)
-		checkClient = logClient
-	}
+	glog.V(1).Infof("Use CT log at %s", uri)
+	logClient, err = client.New(uri, httpClient, opts)
+	checkClient = logClient
```

Comment:
Is this assignment needed, or you can use `logClient` directly instead of `checkClient`?

Reply:
Yes, we can!